civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5015,6 +5015,12 @@ function civihr_employee_portal_user_login(&$edit, $account) {
  * @return bool
  */
 function _civihr_employee_portal_should_do_onboarding($account) {
+  $userWantsToTest = isset($_GET['test_onboarding_wizard']);
```

Comment:
Will the format for this query parameter just be `?test_onboarding_wizard` or will it be `?test_onboarding_wizard=1`? If it's the latter you might want to check the actual value here, not just whether it's set.

Reply:
Without 1 as value to the parameter, Drupal will not set the parameter when making the link, that is the reason for 1. In fact is it sufficient to have the parameter "test_onboarding_wizard" set because 1 doesn't have any meaning, what has meaning is the existence of the parameter. For example there is no "test_onboarding_wizard=0" to not test.

Reply:
I think that it's more common to have query parameters in the form `?a=b&c=d` instead of `?c&d`, what do you think? If that form is more common it should be an easy check to change from `isset` to checking `isset` as well as the value

Reply:
It is an easy change. I will change it now.